.github/workflows/ci.yml

```
@@ -10,11 +10,12 @@ on:
     types: [published]
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  group: ${{ github.ref }}
```

Is there a reason why we would want this workflow to cancel other unrelated workflows?

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow


```suggestion
  group: ${{ github.workflow }}-${{ github.ref }}
```

---

I didn't mean to change this, apologies, reverted!